package.json

```diff
@@ -30,7 +30,7 @@
     "test": "tsc --project . && nyc mocha -r ts-node/register test/unit-tests/index.test.ts && nyc report --reporter=text-summary",
     "integration-test": "node ./node_modules/istanbul/lib/cli.js test _mocha test/integration-tests -- -R spec",
     "lint": "tslint --project .",
-    "build": "tsc --project . && npm publish"
```

Comment:
Do we need the publish command?

Reply:
Nope - Travis runs `npm publish`, that's why I specified just compiling before Travis deploys 
https://github.com/amplify-education/serverless-domain-manager/pull/181/commits/3797bb5e219826171fc9c64e321fc648363430e3#diff-354f30a63fb0907d4ad57269548329e3R14